### ISSUE TITLE
MAX Dynamic Lua improvement request

### ISSUE THREAD

**Issue by AmenMoses** [TheGameCreators/GameGuruRepo#3235]
I would like to request that the '-- DESCRIPTION:' lines in Lua scripts be able to either contain formatting information to enable the editor representation of dynamic Lua parameters can be 'prettyfied' or that as a simple starter each line in '-- DESCRIPTION:' line present in the script be displayed in the editor as a new line.

i.e. instead of:
-- DESCRIPTION: Cine Guru Image trigger. 
-- DESCRIPTION: Trigger [@ImageType=1(1=Image,2=Sequence,3=Letterbox)]
-- DESCRIPTION: Image to use [ImageFile$=""]
-- DESCRIPTION: When to display image [TriggerTime#=0] (seconds)
-- DESCRIPTION: How long to display image [ImageDuration#=0] (seconds)
-- DESCRIPTION: For sequence; time to display each frame [FrameTime=100] (milliseconds)

Being displayed in the editor as:
![dynamiclua](https://user-images.githubusercontent.com/35708338/171502029-a2173589-bfd1-4250-8535-23cf452db811.png)

Have it more readable.

**Comment by Necrym59** [comment]
I agree wholehartedly for this change its very messy with the one single sentence idea. If every --DESCRIPTION creates a new line in the information panel it would be much better. For example in the above Amens example its more readble to the user if it formated out similar to this.
![result2](https://user-images.githubusercontent.com/95015657/171512550-e2fedeae-e38c-4f55-aa1e-2c720738fc89.png)
